cmd/helmitTypes.go

```diff
@@ -21,3 +23,26 @@ type HelmitResponse struct {
 	Error    bool            `json:"error"`
 	Replicas []HelmitReplica `json:"replicas"`
 }
+
+//ShipmentStatus represents the deployed status of a shipment
+type ShipmentStatus struct {
+	Namespace string `json:"namespace"`
+	Version   string `json:"version"`
+	Status    struct {
+		Phase      string `json:"phase"`
+		Containers []struct {
+			Image    string `json:"image"`
+			Ready    bool   `json:"ready"`
+			Restarts int    `json:"restarts"`
+			State    struct {
+				Running struct {
```

Comment:
State may not alway be `Running`. It could be `Waiting`. For example, you may get:
```json
"state": {
  "waiting": {
    "reason": "CrashLoopBackOff",
    "message": "Back-off 5m0s restarting failed container=cnn-now pod=thedeployment-3494639297-303ad_cnn-now-prod(94291f44-6fa4-11e6-ae33-0a72129afe9b)"
  }
}
```
Don't know if that matters or not here.